sal_interfaces/DREAM/DREAM_Commands.xml

```diff
@@ -3,61 +3,98 @@
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>DREAM</Subsystem>
+    <EFDB_Topic>DREAM_command_openHatch</EFDB_Topic>
+    <Description>Open the hatch if DREAM has evaluated that it is safe to do so.</Description>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>DREAM</Subsystem>
+    <EFDB_Topic>DREAM_command_closeHatch</EFDB_Topic>
+    <Description>Close the hatch.</Description>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>DREAM</Subsystem>
+    <EFDB_Topic>DREAM_command_stop</EFDB_Topic>
+    <Description>Immediately stop operations and close the hatch.</Description>
   </SALCommand>
   <SALCommand>
     <Subsystem>DREAM</Subsystem>
-    <EFDB_Topic>DREAM_command_operate</EFDB_Topic>
-    <Description>Operate DREAM, meaning taking images, or not.</Description>
+    <EFDB_Topic>DREAM_command_readyForData</EFDB_Topic>
+    <Description>Inform DREAM that Rubin Observatory is ready to receive data or not.</Description>
     <item>
-      <EFDB_Name>operating</EFDB_Name>
-      <Description>DREAM operating, meaning taking images, or not.</Description>
+      <EFDB_Name>ready</EFDB_Name>
+      <Description>Ready to receive data or not.</Description>
       <IDL_Type>boolean</IDL_Type>
       <Units>unitless</Units>
       <Count>1</Count>
     </item>
   </SALCommand>
   <SALCommand>
     <Subsystem>DREAM</Subsystem>
-    <EFDB_Topic>DREAM_command_status</EFDB_Topic>
-    <Description>Request a full DREAM system status.</Description>
+    <EFDB_Topic>DREAM_command_dataArchived</EFDB_Topic>
+    <Description>Inform DREAM that Rubin Observatory has received and archived a data product. It is still under discussion if one or more parameters are needed or not.</Description>
   </SALCommand>
   <SALCommand>
     <Subsystem>DREAM</Subsystem>
     <EFDB_Topic>DREAM_command_setWeatherInfo</EFDB_Topic>
     <Description>Send the weather information to DREAM.</Description>
     <item>
-      <EFDB_Name>weatherInfo</EFDB_Name>
-      <Description>The weather info for DREAM in JSON format.</Description>
-      <IDL_Type>string</IDL_Type>
-      <Units>unitless</Units>
+      <EFDB_Name>temperature</EFDB_Name>
+      <Description>The current temperature.</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>deg_C</Units>
       <Count>1</Count>
     </item>
-  </SALCommand>
-  <SALCommand>
-    <Subsystem>DREAM</Subsystem>
-    <EFDB_Topic>DREAM_command_getDataProduct</EFDB_Topic>
-    <Description>Request a data product from DREAM.</Description>
     <item>
-      <EFDB_Name>product_name</EFDB_Name>
-      <Description>The name of the DREAM data product.</Description>
-      <!-- DM-29091 TODO: Replace with an enum as soon as the data products have been defined.  -->
-      <IDL_Type>string</IDL_Type>
-      <Units>unitless</Units>
+      <EFDB_Name>humidity</EFDB_Name>
+      <Description>The current humidity.</Description>
+      <IDL_Type>int</IDL_Type>
```

Comment:
Do we need the humidity value to be `float`?

Reply:
The data are coming from the WeatherStation and I checked that XML and indeed it is float. I will modify that.